src/arraycopy.cpp

```diff
@@ -0,0 +1,22 @@
+#include "arraycopy.h"
+#include <iostream>
+#include <fstream>
+
+void arraycopy(const std::string& filename, int arr[], int size) {
```

Comment:
Can we rename this to something like "readFileToArray()" just so it's clearer what it's doing?